backend/src/shared/enums/sort-order.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum SortOrder {
+  Ascending = 'asc',
+  Descending = 'desc',
```

Comment:
Consider make it upper case.

```suggestion
export enum SortOrder {
  ASCENDING = 'asc',
  DESCENDING = 'desc',
}```

Reply:
done